src/theme/components.tsx

```diff
@@ -209,12 +209,15 @@ export function ExternalLinkIcon({
   )
 }
 
-const ToolTipWrapper = styled.div<{ isCopyContractTooltip?: boolean }>`
+const TOOLTIP_WIDTH = 60
+
+const ToolTipWrapper = styled.div<{ isCopyContractTooltip?: boolean; tooltipX?: number }>`
   display: flex;
   flex-direction: column;
   align-items: center;
-  position: ${({ isCopyContractTooltip }) => (isCopyContractTooltip ? 'relative' : 'absolute')};
-  right: ${({ isCopyContractTooltip }) => isCopyContractTooltip && '50%'};
+  position: absolute;
+  left: ${({ isCopyContractTooltip, tooltipX }) =>
+    isCopyContractTooltip && (tooltipX ? `${tooltipX - TOOLTIP_WIDTH / 2}px` : '50%')};
```

Comment:
this seems a lil funky to me. is there any other way to do this?

Reply:
there may be a way to do it on the cursor itself i can look into it

Reply:
i looked into it a while and don't see alternative options. there is no way to position relatively to cursor. i could instead do the calculation outside of the wrapper if we want?